app/models/attachable.rb

```diff
@@ -6,6 +6,7 @@ module Attachable
              -> { not_deleted.order("attachments.ordering, attachments.id") },
              as: :attachable,
              inverse_of: :attachable
+
```

Comment:
Should we be implementing this behaviour in `Attachable`, or should it be implemented directly on `Edition`? 🤔

I don't know the answer, but want to ask the question because I'm not 100% sure it feels right in `Attachable`.

Other non-Edition models can use `Attachable` (such as [Response] and [PolicyGroup]), so we can't necessarily assume that we're operating on a model which follows the [Edition Workflow].

This is reflected in the fact that we've only implemented the UI side of this in the Edition views – because this functionality is specific to Edition change notes.

[Response]: https://github.com/alphagov/whitehall/blob/da1a57c95f323e5d25577b4de231589c09cab3e9/app/models/response.rb#L2
[PolicyGroup]: https://github.com/alphagov/whitehall/blob/86bbd48cfc106ff05960b407fa04f6d7bb234338/app/models/policy_group.rb#L3
[Edition Workflow]: https://github.com/alphagov/whitehall/blob/6a9dab86a24e1fdbed6d508dfbaff96469f19200/app/models/edition/workflow.rb

Reply:
Hmmm, I'm in two minds about this. I think attachable makes sense since there are editions that don't have attachments that don't need these methods. 

But totally agree with what you're saying that the edition.created_at stuff is going to mess up a bit. Not sure what the best place is either, happy to move it to edition, or any other suggestions? New file of some sort or is that adding to the complexity further?

Reply:
I'm sort of on the fence, too. This is probably fine as it is, because Attachable does seem to have some implicit knowledge of Edition (e.g. the `Attachable::Null` class, which seems to emulate an Edition).

I'm tempted to just mark this as resolved and accept it as is.